src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -962,6 +962,37 @@ void* cSTIR_writeImage(void* ptr_i, const char* filename)
 	CATCH;
 }
 
+extern "C"
+void* cSTIR_ImageData_zoom_image(void* ptr_im, const size_t zooms_ptr_raw, const size_t offsets_in_mm_ptr_raw,
```

Comment:
from the .h,seems that `size_t`->`PTR_FLOAT`

Reply:
think it can be anything (if seen `void *` elsewhere) as it goes through a C-style cast.